java/openjdk11/Portfile

```diff
@@ -19,29 +19,55 @@ fetch.type          git
 git.url             https://github.com/openjdk/jdk11u
 git.branch             jdk-${version}-ga
 depends_build       port:autoconf \
-                    port:gmake
-depends_lib         port:openjdk11-bootstrap
+                    port:gmake \
+                    port:bash \
+                    port:openjdk11-bootstrap
+
+if {${configure.build_arch} eq "x86_64"} {
+    patchfiles          
+} elseif {${configure.build_arch} eq "arm64"} {
+    patchfiles          patch-openjdk11-arm64.diff
+    if {${os.major} == 21} {
+        patchfiles-append          patch-openjdk11-build1.diff
+    } elseif {
+        patchfiles-append          
+    }
+    depends_build-replace       port:openjdk11-bootstrap port:openjdk11-zulu
+    configure.pre_args-replace  --with-boot-jdk=/Library/Java/JavaVirtualMachines/openjdk11-bootstrap/Contents/Home \
+                                --with-boot-jdk=/Library/Java/JavaVirtualMachines/openjdk11-zulu/Contents/Home
+    configure.post_args --with-build-jdk=/Library/Java/JavaVirtualMachines/openjdk11-zulu/Contents/Home \
+                        --openjdk-target=aarch64-apple-darwin
+    configure.args-delete      --with-debug-level=release \
```

Comment:
any reason to delete those in case of arm?

Reply:
During building, Azul Zulu is set as build JDK and thus all the argument from Azul Zulu is passed to the build system

Reply:
I can remove `configure.args-delete` to make the build more customisable.

Reply:
btw @usersxx,  I had to move this whole configure section after configure.args (take a look at my diff), otherwise the configure arguments where not being added/replaced.

Reply:
Oops, didn't notice that.